src/app/courses-page/course/course.component.scss

```diff
@@ -115,3 +115,112 @@
   display: flex;
   align-items: center;
 }
+
+.delete-popup {
+  border-radius: 6px;
+	background: color(white);
+	box-shadow: 0 1px 2px 0 rgba(29,30,38,0.10), 0 2px 10px 0 rgba(29,30,38,0.15);
+	height: 217px;
+	width: 370px;
+}
+
+.popup-header {
+  line-height: 24px;
+  letter-spacing: 0.1px;
+  color: color(gunPowder);
+  font-size: 18px;
+  font-family: font(semiBold);
+  text-transform: none;
+  text-align: left;
+  font-weight: 600;
+  height: 60px;
+  padding: 18px 24px;
+  border-bottom: 1px solid color(linkWater);
+  display: flex;
+  justify-content: space-between;
+}
+
+.popup-text {
+  line-height: 24px;
+  letter-spacing: 0.08px;
+  color: color(gunPowder);
+  font-size: 14px;
+  font-family: font(regular);
+  text-transform: none;
+  text-align: left;
+  height: 95px;
+  padding: 19px 24px;
+  border-bottom: 1px solid color(linkWater);
+}
+
+.popup-buttons {
+  padding: 13px 24px;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.popup-btn {
+  padding: 6px 12px;
+  line-height: 24px;
+  letter-spacing: -0.1px;
+  color: color(white);
+  font-size: 14px;
+  font-family: font(semiBold);
+  text-transform: none;
+  font-weight: 600;
+  border-radius: 3px;
+  border: none;
+}
+
+.delete-p-btn {
+  @include change-focus-view(color(oliveShadow));
+  background: color(olive1);
+
+  &:hover {
+    cursor: pointer;
+    background-color: color(lasPalmas);
+  }
+
+  &:active {
+    background-color: color(vidaLoca);
+  }
+}
+
+.cancel-btn {
+  @include change-focus-view(color(whiteShadow));
+  border: 1px solid color(echoBlue);
+  color: color(echoBlue);
+  margin-right: 12px;
+
+  &:hover {
+    cursor: pointer;
+    background-color: color(whiteShadow);
+  }
+
+  &:active {
+    background-color: color(manatee);
+    color: color(white);
+  }
+}
+
+.close-icon {
+  cursor: pointer;
+}
+
+::ng-deep {
```

Comment:
use `GlobalPositionStrategy` for `cdkOverlayOrigin`. By default `cdkOverlayOrigin` uses `FlexibleConnectedPositionStrategy`. You need global to position to put it the center. Those styles will be redundant in that case